pkgs/tools/networking/persepolis/default.nix

```diff
@@ -18,33 +18,26 @@
 
 buildPythonApplication rec {
   pname = "persepolis";
-  version = "3.2.0";
+  version = "4.0.0";
 
   src = fetchFromGitHub {
     owner = "persepolisdm";
     repo = "persepolis";
-    rev = version;
-    sha256 = "1rh7q432ynbysapsd075nif975ync71icpb71x2mb4j8jx1vzs45";
+    rev = "refs/tags/${version}";
+    sha256 = "sha256-2S6s/tWhI9RBFA26jkwxYTGeaok8S8zv/bY+Zr8TOak=";
   };
 
   # see: https://github.com/persepolisdm/persepolis/blob/3.2.0/setup.py#L130
```

Comment:
How are missing dependencies a problem here?

Reply:
The issue was that the tests checked for the presence of some dependencies that weren't found. I now enabled the checkPhase again and instead fixed the tests themselves.